src/translations/pt.json

```diff
@@ -90,6 +90,23 @@
     "driver_master.epod_username.placeholder": "Username",
     "driver_master.epod_password.placeholder": "Password",
 
+    "excel_upload": "Upload do Excel",
+    "excel_upload.about.label": "Este formulário ajuda a carregar dados na ferramenta usando o excel",
+    "excel_upload.added_file.label": "Arquivo adicionado!",
+    "excel_upload.error_upload.header": "Erro!",
+    "excel_upload.error_upload.label": "Não foi possível aplicar as alterações",
+    "excel_upload.error.header": "Erro ao adicionar",
+    "excel_upload.error.label": "Este documento não é compatível, exclua e faça upload de outro arquivo.",
+    "excel_upload.instruction.label": "Arraste e solte o arquivo ou",
+    "excel_upload.instruction.choice": "escolher arquivo",
+    "excel_upload.label": "Upload do Excel",
+    "excel_upload.selection.label": "Selecione uma Mesa",
+    "excel_upload.success.header": "Sucesso!",
+    "excel_upload.success.detail": "Seu arquivo foi carregado com sucesso!",
+    "excel_upload.upload_new_file.label": "Carregar novo arquivo",
+    "excel_upload.upload_try_again.label": "Tente novamente",
+    "excel_upload.view_template": "Ver Modelo",
+
```

Comment:
Something wrong is here - `pt.json` should have the same length what `en.json` but he doesn't

Reply:
fixed